kie-wb-common-dmn/kie-wb-common-dmn-client/src/main/java/org/kie/workbench/common/dmn/client/editors/documentation/links/NameAndUrlPopoverView.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2019 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.kie.workbench.common.dmn.client.editors.documentation.links;
+
+import java.util.function.Consumer;
+
+import org.kie.workbench.common.dmn.api.property.dmn.DMNExternalLink;
+import org.kie.workbench.common.dmn.client.widgets.grid.controls.PopupEditorControls;
+import org.kie.workbench.common.dmn.client.widgets.grid.controls.popover.PopoverView;
+import org.uberfire.client.mvp.UberElement;
+
+public interface NameAndUrlPopoverView extends PopoverView,
+                                               UberElement<NameAndUrlPopoverView.Presenter> {
```

Comment:
We need to use the new `UberElemental` instead of the deprecated `UberElement`.
```suggestion
                                               UberElemental<NameAndUrlPopoverView.Presenter> {
```

Reply:
+1

Reply:
This is not gonna work without a huge refactor, because of `AbstractPopoverViewImpl` that we use to show popups.

Reply:
Perhaps create a JIRA and add it to https://issues.jboss.org/browse/DROOLS-2974

Reply:
Nice. JIRA created. https://issues.jboss.org/browse/DROOLS-4329